lib/logstash/outputs/elasticsearch/common.rb

```diff
@@ -174,8 +174,15 @@ def event_action_params(event)
         params[:pipeline] = event.sprintf(@pipeline)
       end
 
-     if @parent
-        params[:parent] = event.sprintf(@parent)
+      if @parent
```

Comment:
we still support it, as long as you don't use the join_field option, then the parent option behaves as it did before.